### ISSUE TITLE
Problem Occuring When There is No Shot from one of team

### ISSUE THREAD

**Issue by hedonistrh** [oseymour/ScraperFC#9]
Hey, in this match https://fbref.com/en/matches/dfe3f4a0/DC-United-New-York-City-FC-September-6-2020-Major-League-Soccer there is no shot from the home team. 😓 Because of that there can be exception in [here](https://github.com/oseymour/ScraperFC/blob/main/build/lib/ScraperFC/FBRef.py#L967-L969). To solve that, probably, need to check first column's name and then set those stats. 🤔 

I am adding an example if anyone would like to re-produce the issue.

```python
import ScraperFC as sfc
scraper_fbref = sfc.FBRef()
single_match_url = "https://fbref.com/en/matches/dfe3f4a0/DC-United-New-York-City-FC-September-6-2020-Major-League-Soccer"
year = 2020
league_name = "MLS"
fbref_match_data = scraper_fbref.scrape_match(link=single_match_url, year=year, league=league_name)
```

**Comment by oseymour** [comment]
@hedonistrh This should be fixed in the latest version, 2.1.2. Let me know if you're still having problems.

**Comment by hedonistrh** [comment]
Hey @oseymour, thanks a lot for taking care of that bug. Looks like scrape_match() just need _url_ instead of additional _year_ and _league name_. When I make that change and run the following, that is working without any issue. So I am closing that. Thanks again. 🙏🏼 

```python
import ScraperFC as sfc
scraper_fbref = sfc.FBRef()
single_match_url = "https://fbref.com/en/matches/dfe3f4a0/DC-United-New-York-City-FC-September-6-2020-Major-League-Soccer"
year = 2020
league_name = "MLS"
fbref_match_data = scraper_fbref.scrape_match(link=single_match_url)
```
